client/src/components/main/splash/SearchSuggestion.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+
+const Suggestion = styled.div`
+  display: border-box;
+  height: 100%;
+  max-width: 100%;
+  padding: 10px 20px;
+  div {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
+`
+
+export default ({ suggestion: { id, founded, name, address, origin } }) => {
+  const history = useHistory();
+  const dest = (founded ? `/shop/${id}` : `/coffee/${id}`);
```

Comment:
At first I still thought this should point us to the results, but after thinking about how specific the suggestions are in that they point to specific shop locations, I think this is a totally reasonable thing to do.